contributed_definitions/NXem_eds.nxdl.xml

```diff
@@ -72,7 +72,38 @@ and Adrien Teutrie, Cecile Hebert (EPFL)-->
                  The program with which the indexing was performed.
             </doc>
         </group>
-        <group name="PEAK" type="NXpeak">
+        <group name="summary" type="NXdata">
+            <doc>
+                 Accumulated intensity over all pixels of the region-of-interest.
+            </doc>
+            <field name="intensity" type="NX_NUMBER" units="NX_UNITLESS">
+                <doc>
+                     Accumulated counts
+                </doc>
+                <dimensions rank="1">
+                    <dim index="1" value="n_photon_energy"/>
+                </dimensions>
+                <attribute name="long_name" type="NX_CHAR">
+                    <doc>
+                         Counts
+                    </doc>
+                </attribute>
+            </field>
+            <field name="axis_energy" type="NX_NUMBER" units="NX_ENERGY">
+                <doc>
+                     Energy axis
+                </doc>
+                <dimensions rank="1">
+                    <dim index="1" value="n_photon_energy"/>
+                </dimensions>
+                <attribute name="long_name" type="NX_CHAR">
+                    <doc>
+                         Energy
+                    </doc>
+                </attribute>
+            </field>
+        </group>
+        <group type="NXpeak">
```

Comment:
what about a peak_table which collects all peaks into a single group?

Reply:
For now like in MPES and APM peaks as individual groups, indeed argument for a better memory layout of groups N being in the hundreds and thousands (typically not the case for EDX also) is acknowledged but as mentioned in #140 this also demands then an implicit assumption about how the sequence is to be interpreted. These silent assumption could be spelled out via doc strings or attributes in spirit of depends_on although not named like this but this is separate discussion, specifically also for computational geometry. Is a primitive part of another set just because it is stored as a child in another node that is an instance of some primitive ?

Reply:
Also reading/writing performance as well as convenience in exploration (e.g. in h5web) are important consideration. I think by Sept, we would probably need to convert these groups to tables.